website/docs/addon_3dequalizer_admin.md

```diff
@@ -0,0 +1,47 @@
+---
+id: addon_3dequalizer_admin
+title: 3DEqualizer Admin Docs
+sidebar_label: 3DEqualizer
+description: AYON 3DEqualizer Addon's documentations for admins.
+toc_max_heading_level: 5
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.Equalizer_Badge}
+</ReactMarkdown>
+
+## Overview
+
+To get the 3DEqualizer (3DE) addon running, we need to install Qt Python bindings. We use a pre-launch hook that automatically installs `PySide2/6` into 3DE Python. You can set the location using the `TDE4_HOME` environment variable, or it will be deduced from the 3DE executable path in AYON applications settings.
```

Comment:
Hey, should that be `TDE4_ROOT` instead?
reference: [Environment variables in 3DE4](https://www.3dequalizer.com/?site=tech_docs&id=121221_01)
```suggestion
To get the 3DEqualizer (3DE) addon running, we need to install Qt Python bindings. We use a pre-launch hook that automatically installs `PySide2/6` into 3DE Python. You can set the location using the `TDE4_ROOT` environment variable, or it will be deduced from the 3DE executable path in AYON applications settings.
```

Reply:
@antirotor can answer as I dont have the 3DE